manifests/provisioners/helm/resources/deployment.yaml

```diff
@@ -66,6 +70,10 @@ spec:
               mountPath: /var/cache/bundles
             - name: certs
               mountPath: /etc/pki/tls
+          resources:
+            requests:
+              cpu: 10m
+              memory: 160Mi
```

Comment:
I think this is something we can tune over time. What I had landed on for a component that uses rukpak was [100Mi](https://github.com/openshift/platform-operators/blob/main/manifests/0000_50_cluster-platform-operator-manager_04-rukpak-core.deployment.yaml#L78) given that's what OLM is configured to use.

We do store an in-memory bundle cache, which may warrant some benchmarking to get a sense on the right request values here.